updater/src/src/main/java/service/health/check/updater/CheckedAddressHandlerUtil.java

```diff
@@ -0,0 +1,38 @@
+package service.health.check.updater;
+
+import java.sql.Timestamp;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import service.health.check.models.Address;
+
+@Slf4j
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class CheckedAddressHandlerUtil {
+
+    // constants
+    private static final long SECONDS_TO_MILLIS = 1000;
+
+    public static boolean shouldSendMailToFirstAdmin(Address address, Timestamp now) {
+        if (address.getLastHealthy() == null) {
```

Comment:
Dlaczego tak?

Reply:
Potrzebuję więcej szczegółów :P 
jeżeli chodzi o to, czemu nie chcę wysyłać maila jak nie ma health checka, to dlatego, że tak – w sensie, uznałem, że jeżeli dodaliśmy nowy cel i jeszcze ani razu nie udało nam się dostukać, to znaczy, że jeszcze setupujemy tamten cel i dlatego nie jest OK